torch/csrc/jit/codegen/cuda/expr_evaluator.h

```diff
@@ -32,11 +32,12 @@ class TORCH_CUDA_CU_API ExpressionEvaluator : private OptOutDispatch {
   //! Debugging helper, prints all the currently known values
   void print() const;
 
- private:
+ protected:
```

Comment:
let's not open up the guts of the ExpressionEvaluator. If we need additional functionality that can be folded in directly into EE then let's do that instead (see my IterationDomainAnalysis  comments)

If we really need an extensible EE we can design one, but we should have a good justification for it.